File: base/csharp/src/test/csharp/GameOfLifeTest.cs
using csharp.main.csharp;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace csharp.test.csharp {
    [TestClass]
    public class GameOfLifeTest {
        [TestMethod]
        public void TestInitPattern() {
            var classUnderTest = new GameOfLife("2;2;0;1000$");
            Assert.AreEqual("2;2;0;1000$\n", classUnderTest.GetPattern(0));
        }
    }
}

File: gol/java/src/test/java/GameOfLifeTest.java

import org.junit.Test;
import gol.GameOfLife;
import static org.junit.Assert.*;

public class GameOfLifeTest {
	@Test
	public void testInitPattern() {
		GameOfLife classUnderTest = new GameOfLife("2;2;0;1000$");
		assertEquals("2;2;0;1000$\n",classUnderTest.getPattern(0));
	}
}
